src/h.ts

```diff
@@ -1,5 +1,5 @@
-import { vnode, VNode, VNodeData } from './vnode';
-import * as is from './is';
+import { vnode, VNode, VNodeData } from './vnode.js';
+import * as is from './is.js';
```

Comment:
Is this recommended practice, please?

I know that extensionless import paths are non-ES-modules-spec-compliant. This doesn't seem right, though.

Reply:
Can you elaborate? I'm not sure what you mean by not right. Are you saying they should be including `.ts` versions instead?

Reply:
If you're talking about `*` then https://github.com/snabbdom/snabbdom/pull/592#issuecomment-622307219, I think.